lib/longtail/mdnjs/toxml.py

```diff
@@ -0,0 +1,35 @@
+"""
+ Generate a longtail XML dump of the Fathead output.txt
+"""
+import csv
+
+from parse import FatWriter
+
+XML = """<?xml version="1.0" encoding="UTF-8"?>
+<add allowDups="true">
+{results}
+</add>
+"""
+
+DOC = """<doc>
+	<field name="title"><![CDATA[{title}]]></field>
+	<field name="paragraph"><![CDATA[{abstract} <a href="{source_url}">{source_url}</a>]]></field>
```

Comment:
Can you add a `l2_sec` field too. 
```
<field name="title"><![CDATA[{title_strip}]]></field>
```

Then on line 28 make a copy of the title field with punctuation removed. Feel free to implement that however you want. This is just how I did it. 
```
line['title_strip'] = re.sub(r'[^\w\s]',' ',line['title'])
```